colossus/src/main/scala/colossus/core/Connection.scala

```diff
@@ -74,6 +74,11 @@ trait WriteEndpoint {
    * Re-enable reads, if they were previously disabled
    */
   def enableReads()
+
+  /**
+   * Returns a timestamp, in milliseconds of when the last time data was written on the connection
+   */
+  def lastTimeDataWritten: Long
 }
 
 private[core] abstract class Connection(val id: Long, val key: SelectionKey, _channel: SocketChannel, val handler: ConnectionHandler)(implicit val sender: ActorRef)
```

Comment:
Grr..I cannot comment on the line directly, but info function's way of calculating timeIdle needs to change.  It is only accounting for lastTimeDataReceived.  Do we want to also include lastTimeDataWritten?